### ISSUE TITLE
How to use the old logo

### ISSUE THREAD

**Issue by sdaitzman** [VSCodium/vscodium#472]
Is there a way to revert? A setting somewhere? I can also just patch the logo with each update. I really prefer the old one.

**Comment by rublev** [comment]
This is the worst. I'm using default sublime for now lmao.

**Comment by sdaitzman** [comment]
<img width="512" alt="Old Logo" src="https://user-images.githubusercontent.com/1499476/90325153-1cac9a80-df46-11ea-967a-486d523f551b.png">


Old logo for anyone else who comes here looking for it.

**Comment by maelcum** [comment]
You can also copy the old icons file to /Applications/VSCodium.app/Contents/Resources/VSCodium.icns.

This unfortunately does not change all the other times, the logo shows up. Like when you open VSCodium with no files open. Or as tab icon. etc.

I am fully aware that this is an unpopular question, and I therefor ask with all due respect to the awesome people providing us with VSCodium: Is there a way to restore the old icon in all places?

**Comment by sdaitzman** [comment]
@maelcum you should be able to follow @rublev ‘s instructions

**Comment by maelcum** [comment]
@sdaitzman: Changing the application icon of the program shown in Finder is not the problem. That can be done by either @rublev's instructions, or my own as described above. I am talking about the logo showing up in different places. Like when all open tabs are closed. Or when you go to Help > Welcome.

**Comment by sdaitzman** [comment]
Ah, my bad, I misunderstood. I thought you meant that the old icon was still showing up in macOS when referenced at other sizes due to the way iconsets are handled.

Hmm, I think that might require modifying the build of the .app itself. I can try taking a look later!

**Comment by maelcum** [comment]
Thank you!

> when referenced at other sizes due to the way iconsets are handled.
This can be rectified when copying the VSCodium.icns to /Applications/VSCodium.app/Contents/Resources/VSCodium.icns (instead of copy/pasting the finder icon).

**Comment by siffreinsg** [comment]
Is it possible on Windows ?

**Comment by maelcum** [comment]
> Is it possible on Windows ?

For me it works when copying 
C:\Program Files\VSCodium\resources\app\resources\win32\code_70x70.png
and
C:\Program Files\VSCodium\resources\app\resources\win32\code_150x150.png
from an old release to the current release.

**Comment by siffreinsg** [comment]
> > Is it possible on Windows ?
> 
> For me it works when copying
> C:\Program Files\VSCodium\resources\app\resources\win32\code_70x70.png
> and
> C:\Program Files\VSCodium\resources\app\resources\win32\code_150x150.png
> from an old release to the current release.

Thanks!

**Comment by cd4d** [comment]
For Ubuntu 20.04, replacing vscodium.png in `/ush/share/pixmaps` with the old icon (found in this thread) worked.

**Comment by tborychowski** [comment]
old icon wasn't pretty either but at least it fitted nicely with all my apps. This new monstrosity looks like a godzilla trying to eat the mouse pointer before I click it! :-)

**Comment by dg01d** [comment]
For anyone who'd like them, the full set of old icons originally proposed by @jorgeelalto and designed by @jaredreich which created a cohesive and elegant branding for vscodium way back in #4 are in the attached zip file.

![codiums](https://user-images.githubusercontent.com/1061809/91891631-45909780-ec89-11ea-9dc9-3148c0d2869d.png)

[colored.zip](https://github.com/VSCodium/vscodium/files/5158164/colored.zip)

**Comment by brch-cndy** [comment]
Hello, how do I replace the icon on the start page of VSCodium(see pic) with the old one?
![image](https://user-images.githubusercontent.com/62451517/91893730-30504480-ec5a-11ea-8a2c-790fca00870d.png)

**Comment by dg01d** [comment]
Hi @brch-cndy,

That appears to be handled in: 

`workbench/contrib/welcome/code-icon.svg`

I don't know what system you're on (looks like linux?) but on macos this can be changed to the Codium icon by replacing the existing file in `/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/contrib/welcome/code-icon.svg` with the attached file (it needs to be renamed to code-icon.svg)


[code-icon.txt](https://github.com/VSCodium/vscodium/files/5158320/code-icon.txt)

**Comment by brch-cndy** [comment]
I found this directory: /usr/share/vscodium-bin/resources/app/out/vs/workbench/browser/parts/editor/media/ im going to edit the svg you gave me and try it

**Comment by brch-cndy** [comment]
No luck, unfortunately. So sad.

**Comment by jaredreich** [comment]
Old logo files are available in #4. I'm the original creator of them so if you need something else let me know and I may be able to dig up more related files.

I don't get why the logo got changed anyway, this felt pretty solved after #4.

**Comment by dg01d** [comment]
Just wanted to thank you, @jaredreich, the #4 icons are just beautiful and really convey the idea of a Codium.

I don't even know what the new one is supposed to be conveying, perhaps a lack of imagination.

**Comment by cas--** [comment]
@brch-cndy @dg01d The location for the opening logo is `/usr/share/codium/resources/app/out/vs/workbench/browser/parts/editor/media/letterpress-dark.svg`

**Comment by dg01d** [comment]
YES!!!

Thank you @cas--  similar location indeed `/Applications/VSCoodium.app/Contents/Resources/app/out/vs/workbench/browser/parts/editor/media/`

restored the old beautiful & smaller svgs and … it looks *fabulous*

**Comment by chadlavi** [comment]
For anyone else looking, this is the last release with the old icon: https://github.com/VSCodium/vscodium/releases/tag/1.47.2

I was able to use that release to fix the icons on my Mac (these instructions are for Mac only, you'll need to figure it out yourself on other platforms).

1. Download the `.dmg` file from that release and unpack it.
2. There are 5 Svgs in `/Volumes/VSCodium/VSCodium.app/Contents/Resources/app/out/vs/workbench/browser/parts/editor/media`, copy them to `/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/browser/parts/editor/media` : 
```sh
cp -f /Volumes/VSCodium/VSCodium.app/Contents/Resources/app/out/vs/workbench/browser/parts/editor/media/* /Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/browser/parts/editor/media/
```
3. now to change the app icon itself, it's quickest to just use Finder.
    1. right-click the version 1.47.2 VSCodium app in `/Volumes/VSCodium` and click "Get info"
    2. click the app icon in the info window, it should get a focus ring. `cmd-c` to copy it.
    3. Now go right-click on your current version of VSCodium in `/Applications` and click "Get info"
    4. click the app icon, then `cmd-v` to paste the blue codium icon over the new one.
4. If you had VSCodium open, quit and reopen it. This should have fixed the finder/dock/app switcher icon, and also fixed the blank tab background icon.

Happy codi(um)ng!

**Comment by maelcum** [comment]
@chadlavi 
You can skip steps 3 and 4 by copying the icons from the old version to the new, as described in the posts above. Just do a
`copy -f cp -f /Volumes/VSCodium/VSCodium.app/Contents/Resources/*.icns /Applications/VSCodium.app/Contents/Resources/`

This also restores the file type icons.

**Comment by chadlavi** [comment]
@maelcum thanks, maybe I am experiencing a Big Sur dev beta bug. Simply copying over the `.icns` didn't actually change the icon displayed in finder, the dock, or the app switcher for me.

**Comment by maelcum** [comment]
> @maelcum thanks, maybe I am experiencing a Big Sur dev beta bug. Simply copying over the `.icns` didn't actually change the icon displayed in finder, the dock, or the app switcher for me.

Works here on Catalina (Dock, App Switcher, Finder, even the background of an empty VSCodium session). Have no Big Sur to test. Sorry.

**Comment by dg01d** [comment]
Also works on Mojave.

**Comment by chadlavi** [comment]
since @maelcum's solution works on stable macOS releases, here's a simple little repo that can be used to do the swap: https://github.com/chadlavi/vscodium-icon

**Comment by dg01d** [comment]
That is **awesome** @chadlavi. 

I used the dark-green version of the old icon from the #4 icons, because its so pretty.

**Comment by nicovrg** [comment]
I wish one day before updating ui component, company would prompt a poll to user ask to vote for the main logo, then letting them choose which logo version they want to choose, it's so frustrating when you wake up to a day where something you've loved had been replaced by something that's not for your taste ...

**Comment by meduzen** [comment]
> I wish one day before updating ui component, company would prompt a poll to user ask to vote for the main logo, then letting them choose which logo version they want to choose, it's so frustrating when you wake up to a day where something you've loved had been replaced by something that's not for your taste ...

1. VS Codium is not a company.
2. I don’t like the new logo either, so I replaced it by the old one, too.
3. A single search in this repository using the _logo_ keyword brings you to https://github.com/VSCodium/vscodium/issues/491

**Comment by nicovrg** [comment]
> > I wish one day before updating ui component, company would prompt a poll to user ask to vote for the main logo, then letting them choose which logo version they want to choose, it's so frustrating when you wake up to a day where something you've loved had been replaced by something that's not for your taste ...
> 
> 1. VS Codium is not a company.
> 2. I don’t like the new logo either, so I replaced it by the old one, too.
> 3. A single search in this repository using the _logo_ keyword brings you to #491

1. Sorry you're 100% right, vscodium is not a company, that's why i use it btw, i was talking about apps in general 
2. I've replaced the logo, it's not a big deal, i just wish one day we dont need to bother with this and have the choice!

**Comment by stripedpajamas** [comment]
PRs welcome if anyone can figure out a clean way to allow "post build" icon changes easily. 

I'll be closing down this issue since [we've reverted the logo](https://github.com/VSCodium/vscodium/pull/582) back to the classic Codium one (should be in next release). 

I will point out that there's a new vote to update the Codium logo for Big Sur in #591 

To be involved with votes going forward, for now the best approach is to "watch" this repo. In the future we may be able to clobber VSCodium updates (such as links to open votes) onto the existing MS changelog screen that appears after updates.

**Comment by dmoncada** [comment]
Not sure if this is the best place to ask (didn't want to create a new issue), but I was wondering if someone knew where can I find the .icns for the icon with the **black background** that @cristianovitorino created for [this](https://github.com/VSCodium/vscodium/issues/313#issue-544825584) issue (I'm referring to the left-hand-side icon one on option num. 2 in the poll [here](https://github.com/VSCodium/vscodium/issues/313#issuecomment-635612020))? I navigated to commit [46601d6f](https://github.com/VSCodium/vscodium/commit/46601d6f641127762f2ce754883ac2481e170652#diff-4fe51c2eae4f547945188d11577ca7aeda422b4ab3fe3ec3963329e80cc85eb3) but I could only find the .icns for the icon with the light background =/

**Comment by hazarek** [comment]
@dmoncada You can ask a question from this page https://github.com/VSCodium/vscodium/discussions/categories/q-a 
or https://github.com/VSCodium/vscodium/discussions/categories/general.

you can make icns yourself from png;
https://filipmolcik.com/convert-png-to-icns-right-click-converter/

**Comment by ForteDexe** [comment]
Sorry to dig this thread, but why on windows, it is using the old icon, while on linux, it is using the new icon.
